src/pages/white-paper.page.tsx

```diff
@@ -25,40 +27,58 @@ interface WhitePaperPageProps {
 }
 
 export default function WhitePaperPage({ post }): JSX.Element {
+  const contentRef = useRef<HTMLDivElement>(null);
   return (
     <>
-      <Head title={post.title} description={post.description} />
```

Comment:
let's keep the `<Head>`, dont need the description, just the title

Reply:
okie!